src/custom/components/WalletModal/index.tsx

```diff
@@ -10,6 +12,21 @@ export const GpModal = styled(Modal)`
   }
 `
 
-export default function WalletModal(props: Omit<WalletModalProps, 'Modal'>) {
-  return <WalletModalMod {...props} Modal={GpModal} />
+const Blurb = styled.div`
+  width: 100%;
+  margin: 16px 0 0;
+  text-align: center;
+  font-size: smaller;
+  line-height: 1.5;
+`
+
+const NewToEthereum = () => (
+  <Blurb>
+    <div>New to decentralised applications? &nbsp;</div>{' '}
```

Comment:
Double space? `&nbsp;` and `{' '}`

Reply:
was auto by lintyboi

but i think the second can be safely removed

Reply:
And yet, you removed the first :rofl: